cuckoo_time_translator_algorithms/CMakeLists.txt

```diff
@@ -4,12 +4,12 @@ project(cuckoo_time_translator_algorithms)
 find_package(catkin_simple REQUIRED)
 catkin_simple()
 
-find_package(Eigen REQUIRED)
+find_package(Eigen3 REQUIRED)
 find_package(console_bridge REQUIRED)
```

Comment:
This is actually taken care off by catkin simple.

```suggestion
```

I'd be glad if we could do this cleanup with this MR.

Reply:
Sure, I guess I remove `target_link_libraries(${PROJECT_NAME} ${console_bridge_LIBRARIES})` etc. too.

Reply:
Ah, yes, everything with `console_bridge`. Thanks!